net-analyzer/sslsplit/Manifest

```diff
@@ -0,0 +1 @@
+DIST sslsplit-0.5.0.tar.gz 1105325 SHA256 31cb6193124cd42ccb78309fdaa4cd02b8e4809be6778f7066e703e26e8b888a SHA512 a230a1266ed64cd7f01e14b70f5c3300b647c4992c1f36b59456d2e32ff69fefe510f10862d7949dd2e5413833099c439ce72e98a199ea4aac656c4bed362c93 WHIRLPOOL 70bc4f5c4923871f9e19dd42fceb8d1a6c9677c5ad915dde965ddf2dd5b8f1ea5456520b44cf52e8e9f15d8e08c5f4cd1263435d74b00e3e42f4d3dd9af2eab0
```

Comment:
Those are the wrong hashes. Please remove the Manifest file and use `repoman manifest` to regenerate it.